README.md

```diff
@@ -1,289 +1,610 @@
-
-Extra binding properties **SHOULD** also be defined by the bindable service, using one of the patterns defined in [Pointer to binding data](#pointer-to-binding-data).
-
-The data that is injected or mounted into the container may have a different name because of a few reasons:
-* the backing service may have chosen a different name (discouraged, but allowed).
-* a custom name may have been chosen using the `dataMappings` portion of the `ServiceBinding` CR.
-* a prefix may have been added to certain items in `ServiceBinding`, either globally or per service.  
-
-### Request service binding
-
-Binding is requested by the consuming application, or an entity on its behalf such as the [Runtime Component Operator](https://github.com/application-stacks/runtime-component-operator), via a custom resource that is applied in the same cluster where an implementation of this specification resides.
-
-Since the reference implementation for most of this specification is the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) we will be using the `ServiceBinding` CRD, which resides in [this folder](https://github.com/redhat-developer/service-binding-operator/tree/master/deploy/crds), as the entity that holds the binding request.  
-
-**Note** - a few updates / enhancements are being proposed to the current `ServiceBinding` CR, tracked [here](https://github.com/application-stacks/service-binding-specification/issues/16#issuecomment-605309629).
-
-Sample CR
+## Example Directory Structure
+
+```plain
+$SERVICE_BINDING_ROOT
+├── account-database
+│   ├── type
+│   ├── connection-count
+│   ├── uri
+│   ├── username
+│   └── password
+└── transaction-event-stream
+    ├── type
+    ├── connection-count
+    ├── uri
+    ├── certificates
+    └── private-key
 ```
+
+# Service Binding
+
+A Service Binding describes the connection between a [Provisioned Service](#provisioned-service) and an [Application Projection](#application-projection).  It is codified as a concrete resource type.  Multiple Service Bindings can refer to the same service.  Multiple Service Bindings can refer to the same application.
+
+Restricting service binding to resources within the same namespace is strongly **RECOMMENDED**.  Cross-namespace service binding **SHOULD** be secured appropriately by the implementor to prevent attacks like privilege escalation and secret enumeration.
+
+A Service Binding resource **MUST** define a `.spec.application` which is an `ObjectReference` to a `PodSpec`-able resource.  A Service Binding resource **MUST** define a `.spec.service` which is an `ObjectReference` to a Provisioned Service-able resource.  A Service Binding resource **MAY** define a `.spec.name` which is the name of the service when projected into the application.
```

Comment:
FWIW, we use the term "subject" in the Knative Binding duck type.

Our binding shape can be used for non-service binding use cases where the subject of the binding isn't necessarily a PodSpec'able workload though (thus the overly dry "subject" vs. more specific "application")

Reply:
I don't feel strongly about `application`, anybody else have an opinion? The downside to `subject` is that it's not intuitive, one of the first things everybody always asks what subject means.

Reply:
I like `application`, but I have heard the feedback that it's confusing with the SIG Application.  If we changed it in the `ServiceBinding` CR it would be good to change everywhere in the spec.     Not sure what would be a good alternative...`container`?

Reply:
I also don't have a strong opinion on this.  In the CNB spec, we went so far as to remove the word Service because of all of the naming conflicts and ambiguity.  I think that we're always going to have those problems, and being more approachable, even at the risk of being less accurate, has value for gaining adoption this time around.